packages/application-generic/src/services/auth/auth.service.ts

```diff
@@ -201,20 +266,26 @@ export class AuthService {
   }
 
   async generateUserToken(user: UserEntity) {
-    const userActiveOrganizations = await this.organizationRepository.findUserActiveOrganizations(user._id);
+    const userActiveOrganizations =
+      await this.organizationRepository.findUserActiveOrganizations(user._id);
 
     if (userActiveOrganizations && userActiveOrganizations.length) {
       const organizationToSwitch = userActiveOrganizations[0];
 
-      const userActiveProjects = await this.environmentRepository.findOrganizationEnvironments(
-        organizationToSwitch._id
-      );
+      const userActiveProjects =
+        await this.environmentRepository.findOrganizationEnvironments(
+          organizationToSwitch._id
+        );
       let environmentToSwitch = userActiveProjects[0];
 
-      const reduceEnvsToOnlyDevelopment = (prev, current) => (current.name === 'Development' ? current : prev);
+      const reduceEnvsToOnlyDevelopment = (prev, current) =>
+        current.name === 'Development' ? current : prev;
```

Comment:
Is there any possibility to have an enum with environment names or this forces you to do so many changes in the codebase?